### DISCUSSION TITLE
Installazione home Assistant OS che non parte

### DISCUSSION THREAD

**Discussion by eliopei** [indomus/forum#2143] [HUB: Home Assistant]
Ciao a tutti,sono all'inizio con home Assistant,lo avevo installato su Ubuntu ma avevo delle limitazioni perché probabilmente non era una versione supportata,l'ho installato seguendo una guida su indomus usando Ubuntu live da chiavetta e righe di comando, l'installazione mi sembrava andata a buon fine... perlomeno non mi ha dato errori...posto i comandi dati

**Comment by eliopei** [comment]
ubuntu@ubuntu:~$ sudo -i
root@ubuntu:~# sudo apt install curl
Lettura elenco dei pacchetti... Fatto
Generazione albero delle dipendenze      
Lettura informazioni sullo stato... Fatto
I seguenti pacchetti NUOVI saranno installati:
  curl
0 aggiornati, 1 installati, 0 da rimuovere e 0 non aggiornati.
È necessario scaricare 161 kB di archivi.
Dopo quest'operazione, verranno occupati 413 kB di spazio su disco.
Scaricamento di:1 http://security.ubuntu.com/ubuntu focal-security/main amd64 curl amd64 7.68.0-1ubuntu2.14 [161 kB]
Recuperati 161 kB in 1s (120 kB/s)                  
Selezionato il pacchetto curl non precedentemente selezionato.
(Lettura del database... 188529 file e directory attualmente installati.)
Preparativi per estrarre .../curl_7.68.0-1ubuntu2.14_amd64.deb...
Estrazione di curl (7.68.0-1ubuntu2.14)...
Configurazione di curl (7.68.0-1ubuntu2.14)...
Elaborazione dei trigger per man-db (2.9.1-1)...
root@ubuntu:~# curl -L 'https://indomus.it/link/home-assistant-os-x86-64' --output hassio.iso.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
100  351M  100  351M    0     0   639k      0  0:09:23  0:09:23 --:--:--  294k
root@ubuntu:~# unxz hassio.iso.xz
root@ubuntu:~# lsblk
NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
loop0    7:0    0   2,2G  1 loop /rofs
loop1    7:1    0     4K  1 loop /snap/bare/5
loop2    7:2    0  63,2M  1 loop /snap/core20/1695
loop3    7:3    0 346,3M  1 loop /snap/gnome-3-38-2004/119
loop4    7:4    0  91,7M  1 loop /snap/gtk-common-themes/1535
loop5    7:5    0  49,7M  1 loop /snap/snapd/17576
loop6    7:6    0  45,9M  1 loop /snap/snap-store/599
sda      8:0    0 149,1G  0 disk
├─sda1   8:1    0    32M  0 part
├─sda2   8:2    0    24M  0 part
├─sda3   8:3    0   256M  0 part
├─sda4   8:4    0    24M  0 part
├─sda5   8:5    0   256M  0 part
├─sda6   8:6    0     8M  0 part
├─sda7   8:7    0    96M  0 part
└─sda8   8:8    0   1,3G  0 part
sdb      8:16   1  29,8G  0 disk
└─sdb1   8:17   1  29,8G  0 part /cdrom
sr0     11:0    1  1024M  0 rom  
root@ubuntu:~# sudo dd bs=4M if=hassio.iso of=/dev/sda conv=fsync
1536+0 record dentro
1536+0 record fuori
6442450944 bytes (6,4 GB, 6,0 GiB) copied, 136,741 s, 47,1 MB/s
root@ubuntu:~#

**Comment by eliopei** [comment]


**Reply by eliopei** [reply]
Mi dà questo errore (post sotto...non riuscivo a caricare l'immagine da telefono)...non so che pesci prendere...il PC è un dual core da 1,5Ghz 2gb di RAM...un po' datato ma mi sembra che come requisiti minimi ci sia 🤷

**Reply by KC-inDomus** [reply]
io non ho capito qual è l'errore.

**Comment by eliopei** [comment]
![1670356402826](https://user-images.githubusercontent.com/119967475/206012698-499f8c2b-77ec-4639-b83a-c6d511a23a5c.jpg)

**Reply by KC-inDomus** [reply]
o hai sbagliato immagine o non stai facendo boot dalla partizione giusta

**Reply by eliopei** [reply]
https://indomus.it/guide/installare-e-configurare-home-assistant-os-su-un-mini-pc-intel-nuc-dedicato/

ho usato questa guida,fatto partire hubuntu live e poi dato i comandi come qualche post sopra,mi sembra che li sia andato tutto liscio,non mi ha dato errori,però poi non è partito.

**Reply by eliopei** [reply]
non essendo su mini pc non dovevo usare questa guida?

**Reply by KC-inDomus** [reply]
beh, la guida è per mini pc, non per altri hardware specifici...

**Reply by eliopei** [reply]
con le caratteristiche del mio pc un po' datato che configurazione mi consigli?
grazie!!!!

**Reply by KC-inDomus** [reply]
sinceramente non ne ho le minima idea. cmq la soluzione è installare debian 64bit, docker e farci girare o Core o (sigh) Supervised. non certo HassOS come hai provato a fare, secondo me.

**Reply by eliopei** [reply]
Grazie, provo... avevo già provato ma Debian non partiva, cortesemente mi puoi linkare una guida?🤗...così riprovo

**Reply by KC-inDomus** [reply]
https://indomus.it/guide/home-assistant-su-mini-pc-intel-nuc-le-varie-installazioni-le-cose-da-fare/#coredocker
